cni/pkg/ebpf/server/redirectionServer_linux.go

```diff
@@ -131,7 +131,7 @@ func EBPFTProxySupport() bool {
 		return true
 	}
 	if errors.Is(err, ebpf.ErrNotSupported) {
-		log.Infof("FnSkAssign (Linux 5.7 or later) is not supported in current kernel")
+		log.Debugf("FnSkAssign (Linux 5.7 or later) is not supported in current kernel")
```

Comment:
```suggestion
		log.Debugf("FnSkAssign (Linux 5.7 or later) is not supported in current kernel")
```

IMO this should be WARN - it's something you would want to know even if you hadn't explicitly enabled debug logging.